<launch>
    <node name="map_after_process" pkg="map_after_process" type="map_after_process_node" output="screen">
        <param name="map_sub_topic" value="/map_grid_node/map" />
        <param name="map_pub_topic" value="/map_after_process" />
        <param name="radius" value="3"/>
    </node>

</launch>
